code/modules/mob/living/carbon/human/species/necromorph/hunter.dm

```diff
@@ -16,7 +16,6 @@
 */
 #define CUMULATIVE_BURN_DAMAGE	0.5
 #define FAKEDEATH_HEAL_TIME	4 SECONDS
-#define ARM_SWING_RANGE	3
```

Comment:
these are used don't remove them. Give them more specific names  to prevent issues across multiple files

Reply:
Ready